### ISSUE TITLE
Hosted nexushub WebSocket is down: Connecting to the NexusHub API...

### ISSUE THREAD

**Issue by roblabla** [nexus-devs/NexusHub#346]
**I'm submitting a...**

- [x] Bug report
- [ ] Feature request
- [ ] Documentation issue or request

<br>

**Current behavior**
The hosted nexushub seems to be down. When going to https://nexushub.co/wow-classic/ using Firefox and clicking on "Zandalar Tribe" (horde), I just see "Connecting to the NexusHub API...". In the console, firefox complains with various warnings and errors:

```
Firefox can’t establish a connection to the server at wss://auth.nexushub.co/ws.
Source map error: Error: request failed with status 503
Resource URL: https://nexushub.co/bundles/bundle.client.4a4a6ba48db53a79e462.js
Source Map URL: browser.js.map

Firefox can’t establish a connection to the server at wss://api.nexushub.co/ws.
Firefox can’t establish a connection to the server at wss://auth.nexushub.co/ws.
```

Network tab says that requests to `wss://auth.nexushub.co/ws` and `wss://api.nexushub.co/ws` are met with "503 Service Temporary Unavailable".

<br>

**Expected behavior**
To be brought to the server market information page.

<br>

**Minimal reproduction of the problem**
https://nexushub.co/wow-classic/ select Zandalar Tribe, horde side. Any other server/side should work though.

<br>

**Environment**

Browser: 
- [ ] Chrome (desktop) version:
- [ ] Chrome (Android) version:
- [ ] Chrome (iOS) version:
- [x] Firefox version: 75.0a1
- [ ] Safari (desktop) version:
- [ ] Safari (iOS) version:
- [ ] IE version:
- [ ] Edge version:

For development issues:
- Node version:
- Operating System:

##

**Additional Description**

**Comment by Nakroma** [comment]
Hey, I can't reproduce the issue. Can you check if it still occuring for you? Could be a result of the database dowtime a few hours ago.

**Comment by roblabla** [comment]
Alright so I did some testing. Turns out it was caused by some stale settings in a User Agent switcher firefox addon I had installed. Looks like the library used by this website to do websocket breaks when given a different UA than expected. That's a bit scary given U-A are on their way out... But at least I now have a working nexushub! Thanks :D

**Comment by Nakroma** [comment]
Could post the used user agent? That way I can reproduce it and hopefully fix this bug

**Comment by roblabla** [comment]
the UA that breaks it: `Mozilla/5.0 (Windows NT 10.0; WOW64; rv:71.0) Gecko/20100101 Firefox/71.0`
The "Real" UA that fixes it: `Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:75.0) Gecko/20100101 Firefox/75.0`

I'm using https://addons.mozilla.org/en-US/firefox/addon/user-agent-switcher-revived/ to test on Firefox nightly.
